README.md

```diff
@@ -495,7 +495,7 @@ logrus.RegisterExitHandler(handler)
 
 #### Thread safety
 
-By default Logger is protected by mutex for concurrent writes, this mutex is invoked when calling hooks and writing logs.
+By default, Logger is protected by mutex for concurrent writes, and such mutex is invoked when calling hooks and writing logs.
```

Comment:
`By default, Logger is protected by mutex for concurrent writes. The mutex is held when calling hooks and writing logs.`

Reply:
Hi @stevvooe , I like your version better. Just uploaded a new commit 2efdad6. Thanks